Python/1-closure.py

```diff
@@ -0,0 +1,15 @@
+import math
+
+
+def create_log(base):
+    def log(n):
```

Comment:
More direct translation would use lambdas here.
`def` translates as `function` in JavaScript